test/gevent_test.py

```diff
@@ -5,6 +5,9 @@
 
 
 @pytest.mark.skipif(sys.version_info >= (3, 13), reason="gevent seems broken on Python 3.13")
+@pytest.mark.skipif(
+    sys.platform == "win32", reason="gevent support broken on Windows, probably due to event loop patching"
+)
```

Comment:
tbh, not sure how important gevent support is these days, but maybe there are still libraries making use of it?

I feel I'd rather have synchronicity be on the default event loop implementation to prevent potential inconsistencies of using a different one?